package.json

```diff
@@ -20,53 +20,55 @@
   "dependencies": {
     "express": "4.17.1",
     "lodash": "4.17.21",
-    "next": "11.0.1",
-    "next-redux-wrapper": "7.0.2",
+    "next": "12.0.2",
+    "next-redux-wrapper": "7.0.5",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-redux": "7.2.4",
-    "redux": "4.1.0",
-    "redux-thunk": "^2.3.0",
-    "styled-components": "5.3.0",
+    "react-redux": "7.2.6",
+    "redux": "4.1.2",
+    "redux-thunk": "^2.4.0",
+    "sharp": "0.29.2",
+    "styled-components": "5.3.3",
     "typesafe-actions": "5.1.0"
   },
   "devDependencies": {
-    "@babel/plugin-proposal-export-default-from": "7.14.5",
-    "@next/bundle-analyzer": "11.0.1",
+    "@babel/plugin-proposal-export-default-from": "7.16.0",
+    "@next/bundle-analyzer": "12.0.2",
+    "@next/eslint-plugin-next": "12.0.2",
     "@svgr/webpack": "5.5.0",
-    "@types/copy-webpack-plugin": "8.0.0",
-    "@types/express": "4.17.12",
-    "@types/lodash": "4.14.170",
-    "@types/node": "15.12.4",
-    "@types/react": "17.0.11",
-    "@types/react-dom": "17.0.8",
-    "@types/react-redux": "7.1.16",
+    "@types/copy-webpack-plugin": "8.0.1",
+    "@types/express": "4.17.13",
+    "@types/lodash": "4.14.176",
+    "@types/node": "16.11.6",
+    "@types/react": "17.0.33",
+    "@types/react-dom": "17.0.10",
+    "@types/react-redux": "7.1.20",
     "@types/redux": "3.6.31",
-    "@types/styled-components": "5.1.10",
+    "@types/styled-components": "5.1.15",
     "@types/webpack": "5.28.0",
-    "@typescript-eslint/eslint-plugin": "4.28.0",
-    "@typescript-eslint/parser": "4.28.0",
+    "@typescript-eslint/eslint-plugin": "5.2.0",
+    "@typescript-eslint/parser": "5.2.0",
     "babel-plugin-lodash": "3.3.4",
-    "babel-plugin-styled-components": "1.12.0",
+    "babel-plugin-styled-components": "1.13.3",
     "chokidar": "3.5.2",
-    "copy-webpack-plugin": "9.0.0",
-    "esbuild": "0.12.9",
-    "eslint": "7.29.0",
-    "eslint-config-next": "11.0.1",
+    "copy-webpack-plugin": "9.0.1",
+    "esbuild": "0.13.12",
```

Comment:
Just curious; Why is this project both `esbuild` and `webpack`?

Reply:
Esbuild was added to compile a couple of things to JS
- The next.config.ts file. Next does not support a .ts config file out of the box.
- The custom Next server

Both these things would eat up seconds of startup time when building with any other bundler so esbuild is included to compile both these things in milliseconds.

Webpack is used for the Next config.